docs/user/water-bodies.rst

```diff
@@ -1,18 +1,21 @@
 .. _water-bodies:
 
-Water Bodies
-============
+Oceans, Rivers and Lakes
+========================
 
-.. youtube:: jXphUy__J0o
```

Comment:
Removed because it uses water bodies in a legacy way, and uses the now removed Wizard. Need to record a new one ideally.